kbl2vec/src/test/java/com/foursoft/harness/kbl2vec/__snapshots__/KblToVecConverterTest.snap

```diff
@@ -1859,6 +1867,66 @@ This is not intended for productive use and is not complete!
         </Path>
       </Routing>
     </Specification>
+    <Specification xsi:type="vec:PlacementSpecification" id="PlacementSpecification_id_328_0">
+      <Identification>PLACEMENT</Identification>
+      <Placement xsi:type="vec:OnWayPlacement" id="OnWayPlacement_Protection_area_1">
+        <PlacedElement>PlaceableElementRole_Wire_protection_occurrence_1</PlacedElement>
```

Comment:
Forgot one thing: The placement has a mandatory identification. Take the "Id" of the associated wire_protection_occurrence" . This would show up in the schema validation test.